lib/functions.php

```diff
@@ -80,6 +77,9 @@ function html_email_handler_send_email(array $options = null) {
 		$options["bcc"] = array($options["bcc"]);
 	}
 	
+	$options['html_message'] = html_email_handler_make_html_body($options);
+	$options['plaintext_message'] = $options['body'];
```

Comment:
I guess these two should be added only if both of them are empty? Should I also check whether the global HTML email setting has been enabled?